src/tcp_server.rs

```diff
@@ -196,6 +196,19 @@ impl TcpServer {
                                                         }
                                                     }
                                                 }
+                                                ClientMessage::RequestCurrentLayerName {} => {
+                                                    let mut k = kanata.lock();
+                                                    let cur_layer = k.layout.bm().current_layer();
+                                                    let msg = ServerMessage::CurrentLayerName {
+                                                        name: k.layer_info[cur_layer].name.clone(),
+                                                    };
```

Comment:
Same feedback here as on the other PR: it is better to explicitly drop `k` earlier